x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -45,7 +55,22 @@ starting each node.
 
 Apart from setting `xpack.security.fips_mode.enabled`, a number of security
 related settings need to be configured accordingly in order to be compliant
-and able to run {es} successfully in a FIPS 140-2 enabled JVM.
+and able to run {es} successfully in a FIPS 140-2 configured JVM.
+
+[discrete]
+[[keystore-fips-password]]
+==== Elasticsearch Keystore
+
+FIPS 140-2 (via NIST Special Publication 800-132) dictates that encryption keys should at
+least have an effective strength of 112 bits.
+As such, the elasticsearch keystore that stores the node's <<secure-settings,secure settings>>
+needs to be password protected with a password that satisfies this requirement.
+This means that the password needs to be 14 bytes long which is equivalent
+to a 14 character ASCII encoded password, or a 7 character UTF-8 encoded password.
+You can use the <<elasticsearch-keystore, elasticsearch-keystore passwd>> subcommand to change or set the
+password of an existing keystore.
+Note that when the keystore is password-protected, you must supply the password each time
+Elasticsearch starts.
```

Comment:
It doesn't seem like the NIST Special Publication 800-132 is needed.
```suggestion
FIPS 140-2 states encryption keys must have an effective strength of at least
112 bits. To satisfy this requirement, the {es} keystore password must be at
least 14 bytes long, equivalent to a 14-character ASCII password or a
7-character UTF-8 password. You can use the
<<elasticsearch-keystore,`elasticsearch-keystore passwd`>> subcommand to set or
change an existing keystore's password. You must provide this password each time
{es} starts.
```

Reply:
> It doesn't seem like the NIST Special Publication 800-132 is needed.

Why do you say so ?